autogluon/buildspec-0-4-2.yml

```diff
@@ -0,0 +1,89 @@
+account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
```

Comment:
This file is just moved old version of 0.4.2 buildspec. No changes since the last container release.